warehouse/utils/utils.go

```diff
@@ -612,26 +621,36 @@ func Datatype(in interface{}) string {
 	return "string"
 }
 
-//ToSafeDBString to remove special characters
-func ToSafeDBString(provider string, str string) string {
-	res := ""
-	if str != "" {
-		r := []rune(str)
-		_, err := strconv.ParseInt(string(r[0]), 10, 64)
-		if err == nil {
-			str = "_" + str
+func ToSafeNamespace(provider string, name string) string {
```

Comment:
Let's add a comment on what this function does with a few examples.

Reply:
added